src/Common/Core/Impl/Microsoft.R.Common.Core.csproj

```diff
@@ -8,6 +8,7 @@
   <Import Project="..\..\..\R.Settings.props" />
   <ItemGroup>
     <Compile Include="$(SourceDirectory)GlobalAssemblyInfo.cs" Link="Properties\GlobalAssemblyInfo.cs" />
+    <Compile Include="..\..\..\AssemblyVersionInfo.cs" Link="Properties\AssemblyVersionInfo.cs" />
```

Comment:
I'll replace manually. This generated part. Currently we generate for every assembly but it is really unnecessary. Also, adding it to `Compile` works in regular project but produces `File specified twice` in .NET Core since latter picks up all files by default.